src/resources/basis.js

```diff
@@ -0,0 +1,349 @@
+Object.assign(pc, function () {
+
+    var BasisWorker = function (worker, jsUrl, binaryUrl, asmUrl, formats) {
+
+        // transcode the basis super-compressed data into one of the runtime gpu native formats
+        var transcode = function (basis, url, data) {
+            var funcStart = performance.now();
+            var basisFile = new basis.BasisFile(new Uint8Array(data));
+
+            var width = basisFile.getImageWidth(0, 0);
+            var height = basisFile.getImageHeight(0, 0);
+            var images = basisFile.getNumImages();
+            var levels = basisFile.getNumLevels(0);
+            var hasAlpha = !!basisFile.getHasAlpha();
+
+            if (!width || !height || !images || !levels) {
+                basisFile.close();
+                basisFile.delete();
+                throw new Error('Invalid image dimensions');
```

Comment:
It would be helpful to mention that these errors are related to basis

Reply:
Sure, I'll add the url to the message